parsers/imageAltTag.js

```diff
@@ -0,0 +1,47 @@
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('parser:imageAltTag');
+var parse = require('./lib/parse');
+
+/*
+ * Extract alt text from img tags
+ */
+
+var PARSER_COMPLETED_SUCCESSFULLY = false;
+var FACEBOOK_ALT_TEXT_START = 'Image may contain:';
+
```

Comment:
Would be better if the AI interpretation was delegated to a separate parser?
Personally I find this a valuable information to extract, but I agree with you about the implementation, was too simplicistic. Maybe a separate parser with a more accurate heuristic for detecting AI would be a better solution?